Presscotts_Mortar_Calculator_(no_envirement)_V0.2.py

```diff
@@ -19,215 +18,225 @@
    air density 1pct incline]
 '''
 
-import math 
+import math
+
+
 def getList(dict):

```

Comment:
I suspect this function could actually just boil down to `return dict.keys()`.

Reply:
Not exactly 
It puts the result in the form of a list which is used to determine values via indecencies

Reply:
A bit of tomato tomahto, but from the [python docs](https://docs.python.org/3/tutorial/datastructures.html):

> Performing list(d) on a dictionary returns a list of all the keys used in the dictionary, in insertion order (if you want it sorted, just use sorted(d) instead). To check whether a single key is in the dictionary, use the in keyword.

A good general principle is to always prefer built-in methods where available vs. creating and maintaining your own. You likely want `list(dict)` instead.

`dict.keys()` also works but I was a bit mistaken about the return type. Rather than a `list` like it used to return, modern python versions return a "[set-like view](https://docs.python.org/3.8/library/stdtypes.html#dict-views)" which is actually really nice for a couple of reasons. From the doc:

> [`dictview`s] provide a dynamic view on the dictionary’s entries, which means that when the dictionary changes, the view reflects these changes.

So on top of the nice auto-tracking of the underlying object we also avoid making a copy of the dict's data. Copies in memory are one of the most computationally expensive (read: time-consuming) operations and we want to avoid them if we can.

Since we're dealing with small dictionaries the difference between copying and non-copying implementations here will be negligible, but if you're ever curious about this try writing a little benchmark comparing the two over very large dictionaries!